package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@ampproject/animations": "0.1.1",
     "@ampproject/worker-dom": "0.2.17",
+    "amphtml-validator": "^1.0.23",
```

Comment:
This should be in `devDependencies` and pinned to a version (no `^`).

Reply:
Thanks, #20742.